storage/diemdb/src/lib.rs

```diff
@@ -506,6 +557,27 @@ impl DiemDB {
             pruner.wake(latest_version)
         }
     }
+
+    fn update_rocksdb_properties(db: &DB) -> Result<()> {
+        for cf_name in Self::column_families() {
```

Comment:
Please move the reporting of DIEM_STORAGE_CF_SIZE_BYTES over to this thread as well.